corehq/ex-submodules/pillowtop/processors/elastic.py

```diff
@@ -97,26 +93,19 @@ def process_change(self, change):
                 self._delete_doc_if_exists(change.id)
                 return
 
-            # prepare doc for es
-            doc_ready_to_save = self.doc_transform_fn(doc)
```

Comment:
This will change the datadog timing data since the 'transform' step now happens under 'load'.

I suspect that's not and issue and we could potentially remove these metrics @calellowitz